### ISSUE TITLE
Plugin yields no error for files which contain errors

### ISSUE THREAD

**Issue by lufte** [sophilabs/gulp-flake8#1]
Using the latest version of this plugin yields no errors for a python file for which `flake8` yields 2 errors.

The python file contains the following:
```
from datetime import date

print('aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa')
```
Running the native flake8 tool yields:
```
file.py:1:1: F401 'datetime.date' imported but unused
file.py:3:1: T003 print function found.
file.py:3:80: E501 line too long (95 > 79 characters)
```
But running flake8 from a gulp task using this plugin yields no errors. I'm using the sample `gulpfile.js` from the README.md.

**Comment by rlopezcc** [comment]
This seems to be the issue https://github.com/gforcada/flake8-pep3101/issues/15

It can be fixed by installing flake8-pep3101 >= 1.1

Anyway we need to make gulp-flake8 aware of this kind of errors.

**Comment by rlopezcc** [comment]
This was fixed in 18361484a2693438f8d0d76363bb46bcad928463.
